instrumentation/opentelemetry-instrumentation-aiohttp-server/src/opentelemetry/instrumentation/aiohttp_server/__init__.py

```diff
@@ -213,7 +213,7 @@ async def middleware(request, handler):
     active_requests_counter = meter.create_up_down_counter(
         name=MetricInstruments.HTTP_SERVER_ACTIVE_REQUESTS,
         unit="requests",
-        description="measures the number of concurrent HTTP requests those are currently in flight",
+        description="measures the number of concurrent HTTP requests that are currently in-flight",
```

Comment:
yep, that would be great! if you see any issues/opportunities to improve, lmk, happy to update helpers.

Reply:
i'll try to do that. i haven't seen it actually used anywhere yet.

Reply:
That's right, we are using the instrument creation factories now for the migration to the new semconv. At this point, we didn't include aiohttp-server in the list of instrumentation to migrate to the new semconv, but I think it's important since we can release the package now.